src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+class RightTriangle extends Figure {
```

Comment:
The class name 'RightTriangle' is appropriate as it clearly describes the entity it represents. However, ensure that the class does not start with an empty line as per the checklist.